tests/utils/secrets.go

```diff
@@ -0,0 +1,61 @@
+package utils
+
+import (
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"kubevirt.io/containerized-data-importer/pkg/common"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"github.com/golang/glog"
+	"github.com/pkg/errors"
+)
+
+type cdiSecret struct {
+	sec *v1.Secret
+}
+
+type CdiSecret interface {
+	Create(clientset *kubernetes.Clientset) (*v1.Secret, error)
+	Get() *v1.Secret
+}
+
+func NewSecretDefinition(ns, prefix string) *cdiSecret {
+	return &cdiSecret{
+		&v1.Secret{
+			ObjectMeta: metav1.ObjectMeta{
+				GenerateName: prefix,
+				Namespace:    ns,
+				Labels: map[string]string{
+					common.CDI_LABEL_KEY: common.CDI_LABEL_VALUE,
+				},
+			},
+			StringData: map[string]string{
+				"MINIO_ACCESS_KEY":        "admin",
+				"MINIO_SECRET_ACCESS_KEY": "password",
+			},
+		},
+	}
+}
+
+func (s *cdiSecret) Create(c *kubernetes.Clientset) (*v1.Secret, error) {
+	var sec *v1.Secret
+	err := wait.PollImmediate(defaultPollInterval, defaultPollInterval, func() (done bool, err error) {
+		sec, err = c.CoreV1().Secrets(s.sec.Namespace).Create(s.sec)
```

Comment:
do you need to declare `err` before this line?

Reply:
`err` is declared for the subscope as a named return